e2e/po/contact-list.po.ts

```diff
@@ -28,6 +28,17 @@ export class ContactListPageObject extends PageObject {
     this.browser.get('/');
   }
 
+  clickContactNumber(contactNumber: number) {
+    return this.trs.count().then(count => {
+      console.log(count);
```

Comment:
should we remove this console log?

Reply:
Yup, thanks for the review. Still WIP...this is being removed.